### DISCUSSION TITLE
:GSOC can we use the given repo functions in our code

### DISCUSSION THREAD

**Discussion by pyther-hub** [openvinotoolkit/openvino#16231] [Google Summer of Code]
If the repository from which I have cloned the PyTorch model includes a prebuilt method to convert it into ONNX format, then I can use this method to carry out the conversion process.

**Comment by adrianboguszewski** [comment]
@pyther-hub, yes. Please look at 220 or 230 yolo notebooks. They both use a prebuilt method for conversion.
